tests/utils.py

```diff
@@ -181,9 +195,71 @@ def divide_by_int(dividend: Coordinate, divisor: int) -> Coordinate:
             else dividend / divisor)
 
 
+def compound_to_compound_with_multipoint(compound: Compound
+                                         ) -> Tuple[Compound, Multipoint]:
+    return (compound, Multipoint(list(to_unique_ever_seen(compound_to_points(
+            compound)))))
+
+
+def compound_to_points(compound: Compound) -> Iterable[Point]:
```

Comment:
Why not use `functools.singledispatch` here?

Reply:
Because having a single function is easier than multiple ones, since this function is not designed to be used outside of tests, singledispatch is for extending functionality, we don't need this flexibility here